packages/devp2p/test/integration/rlpx-simulator.spec.ts

```diff
@@ -6,94 +6,100 @@ import { DISCONNECT_REASON } from '../../src/types.js'
 import * as util from './util.js'
 
 describe('RLPx simulator tests', () => {
-  it('RLPX: add working node', () => {
+  it('RLPX: add working node', async () => {
     const basePort = 40404
-    const rlpxs = util.initTwoPeerRLPXSetup(undefined, undefined, undefined, basePort)
-
-    rlpxs[0].events.on('peer:added', async (peer) => {
-      assert.equal(
-        peer['_port'],
-        basePort + 1,
-        'should have added peer on peer:added after successful handshake',
-      )
-      assert.equal(rlpxs[0].getPeers().length, 1, 'peer list length should be 1')
-      assert.equal(rlpxs[0]._getOpenSlots(), 9, 'should have maxPeers - 1 open slots left')
-      await util.delay(500)
-      util.destroyRLPXs(rlpxs)
+    const { rlpxs, peer } = util.initTwoPeerRLPXSetup(undefined, undefined, undefined, basePort + 1)
```

Comment:
I believe `basePort + 1` is correct here since the following assert is checking if the peer's port is connecting as expected.